packages/databricks-sdk-js/src/config/Config.ts

```diff
@@ -61,7 +61,7 @@ export type AuthType =
     | "azure-cli"
     | "google-id"
     | "metadata-service"
-    | "bricks-cli";
```

Comment:
@nfx @pietern we need to make this change consistently. How did you name it in go?

Reply:
Can you revert just the auth type name? Then we unblock merging this and we do it in a patch bump.